src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/ContentItemWhereInput.cs

```diff
@@ -36,12 +37,22 @@ public ContentItemWhereInput(string contentItemName, IOptions<GraphQLContentOpti
             Field<ListGraphType<ContentItemWhereInput>>("Not").Description("NOT logical operation").Type(whereInputType);
         }
 
-        private void AddFilterField<T>(string name, string description)
+        public void AddInputField(FieldType fieldType)
+        {
+            AddFilterField(fieldType.Type, fieldType.Name, fieldType.Description);
+        }
+
+        private void AddFilterField<TGraphType>(string name, string description)
+        {
+            AddFilterField(typeof(TGraphType), name, description);
+        }
+
+        private void AddFilterField(Type graphType, string name, string description)
         {
             if (!_optionsAccessor.Value.ShouldSkip(typeof(ContentItemType), name))
             {
-                AddScalarFilterFields<T>(name, description);
+                AddScalarFilterFields(graphType, name, description);
             }
         }
     }
-}
+}
```

Comment:
Add a blank line at the end of the file
```suggestion
}

```